lib/bandit/http2/connection.ex

```diff
@@ -51,27 +51,35 @@ defmodule Bandit.HTTP2.Connection do
         }
 
   @spec init(Socket.t(), Bandit.Pipeline.plug_def(), keyword()) :: {:ok, t()} | no_return()
-  def init(socket, plug, opts) do
+  def init(socket, plug, opts, initial_request \\ nil, remote_settings_payload \\ <<>>) do
```

Comment:
There's a dialyzer failure in CI on this function

Reply:
Sorry forgot about the spec for this function. Fixed in 5152ee0